typescript/integrations.d.ts

```diff
@@ -150,6 +150,15 @@ export type IntegrationFreshdesk = IntegrationBase & {
   }
 }
 
+export type IntegrationMailGun = IntegrationBase & {
+  type: 'MAILGUN'
+  configuration: {
+    baseUrl: string
```

Comment:
If we are going to store just the domain, can we change this property to `domain`?